core/tests/protractor_utils/AdminPage.js

```diff
@@ -93,10 +93,10 @@ var AdminPage = function() {
     };
 
     this.reloadCollection = async function(collectionId) {
-      this.get();
-      await (
-        await reloadCollectionButtons.get(collectionId)
-      ).click();
+      await this.get();
+      var reloadCollectionButton = await (
+        reloadCollectionButtons.get(collectionId));
+      await action.click('reloadCollectionButtons', reloadCollectionButton);
```

Comment:
Let's make the first param more descriptive. e.g.

https://github.com/oppia/oppia/blob/fc48b2c1cd295a8790247aad3ec52896f5741db0/core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js#L150-L152